packages/cli/src/commands/info/index.ts.ts

```diff
@@ -0,0 +1,9 @@
+import { Command } from "commander";
+import { infoAction } from "./action.js";
+import { addFilterOptions } from "../addFilterOptions.js";
+
+const infoCommand = new Command("info");
```

Comment:
Please add a description here - something like this:

"(UNSTABLE) The info command displays information about a target graph in a workspace. The generated output can be read by other task runners, such as BuildXL."